src/modules/Layout/Container/Container.jsx

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
```

Comment:
We need to align `SomeComponent` in the middle (on X axis):
Variant **A**:
`
<Container>
    <SomeComponent ... />
</Container>
`
Variant **B**:
change styles inside `SomeComponent.css`.

Variant **C**:
pass an additional className to `<SomeComponent />`

**Next step**: we need to change alignment just of `<SomeComponent>`:
- In case **A** we just need to remove `<Container>` wrapper or substitute with some another layout alignment component.
- In case **B** we need to rewrite styles inside `SomeComponent`
- in case **C** we need to remove that additional className and pass some another.

In my humble opinion, it's less painful and risky variant **A** approach.
What do you think about this, @nik-kor ?